modules/bbp-atlas/src/test/resources/invalid/bbp/atlas/atlasspatialreferencesystem/v0.1.0/wrong-type.json

```diff
@@ -0,0 +1,53 @@
+{
+  "@context": [
+    "http://localhost/v0/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/"
+    }
+  ],
+  "@type": "nsg:Vector3D",
```

Comment:
Should we stick to **nsg:WrongType** to keep it consistent with other invalid instances?

Reply:
Nice catch. Agreed

Reply:
Thinking about it...
 It's important here to check that using a class targeted in an inherited shape (Vector3DShape) will not pass the validation. So I'll keep the nsg:Vector3D as a type.

Reply:
Good point!